server/services/caseloadService.ts

```diff
@@ -68,9 +68,9 @@ export default class CaseloadService {
       .then(caseload => this.mapResponsibleComsToCases(caseload))
   }
 
-  async getOmuCaseload(user: User, prisonCaseload: string[]): Promise<ManagedCase[]> {
-    // Get cases with a licence in ACTIVE, APPROVED, or SUBMITTED state
-    const casesWithLicences = this.licenceService
+  async getOmuCaseload(user: User, prisonCaseload: string[], view: string): Promise<ManagedCase[]> {
+    // Get cases with a licence in ACTIVE, APPROVED, SUBMITTED, IN_PROGRESS or VARIATION_IN_* state
+    const casesWithLicences = await this.licenceService
```

Comment:
NOT_STARTED status does not exist in the licences DB which means the NOT_STARTED licences are obtained via the call to prisonerService.searchPrisonersByReleaseDate on L81
This already filters out those with release dates going past 4 weeks in future